README.md

```diff
@@ -62,43 +94,67 @@ This is a milestone project which means features will be added in phases.
   </ul>
 </details>
 
+<!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
 <ul>
-<li>Represent my recent works</li>
-<li>Represent my skills and used technologies</li>
-<li>My contact info😎</li>
-<li>Mobile and desktop responsiveness</li>
+<li>[Represent my recent works]</li>
+<li>[Represent my skills and used technologies]</li>
+<li>[My contact info😎]</li>
+<li>[Mobile and desktop responsiveness]</li>
 </ul> 
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## 🚀 Live Demo 
+<!-- LIVE DEMO -->
 
-N/A
+## 🚀 Live Demo 
+https://paulinelali.github.io/potfoilo.github.io/
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
-To get a local copy up and running, follow these steps.
+
 ### Prerequisites
 
 In order to run this project you need:
 To clone or fork and run it in a browser
 
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
 ### Setup
 
-Clone this repository to your desired folder on your local machine:
+Clone this repository to your desired folder:
+
+
+
 
-Example:
 ```sh
   cd my-folder
-  git clone https://github.com/Paulinelali/microverse-personal-portfolio
+  git clone https://github.com/Paulinelali/hello-microverse-project
 ```
 
+### Install
+
+Install this project with:
+
+
+```sh
+  cd my-project
+  gem install
+```
+
+
```

Comment:
- [ ] kindly consider customizing this sub-section for them to reflect exactly the current repo and branch  you are working on to avoid confusion to the person who want to have a copy of your project on their machine